internal/handler/wallet/sign.go

```diff
@@ -98,12 +98,12 @@ func sign(wlt *wallet.Wallet, params operations.RestWalletSignOperationParams) (
 	return pubKey, signature, nil
 }
 
-func handleWithCorrelationId(wlt *wallet.Wallet, params operations.RestWalletSignOperationParams, gc gcache.Cache) (models.CorrelationID, middleware.Responder) {
+func handleWithCorrelationId(wlt *wallet.Wallet, params operations.RestAccountSignOperationParams, gc gcache.Cache) (models.CorrelationID, middleware.Responder) {
 	cacheKey := getCacheKey(params.Body.CorrelationID)
 
 	value, err := gc.Get(cacheKey)
 	if err != nil {
-		return nil, operations.NewRestWalletSignOperationInternalServerError().WithPayload(
+		return nil, operations.NewRestAccountSignOperationInternalServerError().WithPayload(
```

Comment:
Do you get an error if the key is not in the cache ? In that case it's not evrytime an internal error

Reply:
we can check what error happen: `if err == KeyNotFoundError` but maybe it's out of scope of this PR.

Overall I'd return 404, wdty?

Reply:
Yes, it's out of scope. But you should create an issue on that. A 404 sounds ok.

Reply:
#122